### ISSUE TITLE
Contact details

### ISSUE THREAD

**Issue by thatrandomkid45682** [UnamSanctam/SilentCryptoMiner#1754]
Hello Unam, I've been using the SCM for a while now and have been very impressed with your open source product. I view you as a super talented engineer and would love to get in contact with you about something else. Could you be kind enough to give me your email please?

Thanks.

**Comment by Sayyed090** [comment]
please help me i have been infected with this tool...i ran python script from another github repo which said it will increase my youtube views but nothing happend. but then i noticed after evrystart a python script pops up i tried to read ,i dont know any coding, but managed to get here. python code conatined      "https://stromrechnung.ru/miner.exe"  PLEASE HELP ME REMOVE THIS MINER

**Comment by UnamSanctam** [comment]
>Hello Unam, I've been using the SCM for a while now and have been very impressed with your open source product. I view you as a super talented engineer and would love to get in contact with you about something else. Could you be kind enough to give me your email please?

Of course, this is my email unams@protonmail.com. Currently I'm working on the finishing touches of the next miner version so if I don't respond quickly then it's because of that.

>please help me i have been infected with this tool...i ran python script from another github repo which said it will increase my youtube views but nothing happend. but then i noticed after evrystart a python script pops up i tried to read ,i dont know any coding, but managed to get here. python code conatined "https://stromrechnung.ru/miner.exe" "https://stromrechnung.ru/" PLEASE HELP ME REMOVE THIS MINER

I have responded to your problem in your separate issue so please read that whenever you have the time.
